Paxifax.cs

```diff
@@ -21,13 +21,186 @@
 using System.Reflection;
 using System.Linq;
 
+// FIXME use javadoc-style comments to describe the API
 namespace Pax {
 
-  // FIXME use javadoc-style comments to describe the API
-  public interface PacketProcessor {
+  // A more abstract interface to packet processors.
+  public interface Abstract_PacketProcessor {
```

Comment:
Very sensible. I'll update the names, thanks.

On 2016-07-12 14:25, TMVector wrote:
> In Paxifax.cs [1]:
> 
>> namespace Pax {
>> 
>> - // FIXME use javadoc-style comments to describe the API
>> - public interface PacketProcessor {
>> + // A more abstract interface to packet processors.
>> + public interface Abstract_PacketProcessor {
> 
> I don't know what you think about this, but a common pattern is to
> name interfaces e.g. IPacketProcessor.
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/7#discussion_r70436683
> [2]
> https://github.com/niksu/pax/pull/7/files/3d69508e501b5f605e634c939d226987d76f81e6#r70436683
> [3]
> https://github.com/notifications/unsubscribe/ABDTTg_0qVBnlnrO0wjHgRiKHy-vx_2Oks5qU5W_gaJpZM4JKIVi

-- 
http://www.cl.cam.ac.uk/~ns441/